components/bl-stopwatch/styles/index.less

```diff
@@ -1,9 +1,14 @@
 @bl-customComponent-stopwatch-font-size: 16px;
 @bl-customComponent-stopwatch-text-color: @appTextColor;
+@bl-customComponent-stopwatch-font-family: "Lucida Console", monospace;
 
 .bl-customComponent-stopwatch {
```

Comment:
It seems to me that there is no need to separate styles; span will inherit from the parent.

```
.bl-customComponent-stopwatch {
  display: flex;
  font-size: @bl-customComponent-stopwatch-font-size;
  color: @bl-customComponent-stopwatch-text-color;
  font-family: @bl-customComponent-stopwatch-font-family;
}
```